src/sweetalert2.js

```diff
@@ -87,7 +87,9 @@ const setParameters = (params) => {
     title.innerHTML = params.title.split('\n').join('<br />')
   }
 
-  if (!params.backdrop) {
+  if (params.backdrop && typeof params.backdrop === 'string') {
+    dom.getContainer().style.backgroundColor = params.backdrop
```

Comment:
Should we set `style.background` instead of `style.backgroundColor`? It would give the ability to use animated unicorns as a backdrop. Or, for example, the color + the logo in a corner.

Reply:
So it would accept something like `url(../some/image/path)` as well? I like it.